docs/contributing/hack.md

```diff
@@ -15,13 +15,52 @@ Ready to contribute? Here's how to set up *Watson* for local development.
         $ cd Watson
         $ pip install -r requirements-dev.txt
         $ python setup.py develop
+    
+    A few notes:
+
+      - Beware that, depending on your configuration, your virtual environment
+        may be created in a centralized location (like `$HOME/.virtualenvs`),
+        and not in the current folder.
+      
+      - Also, beware that changing the names of the folders pointing to this
+        development folder may give an undesired behavior.
+      
+      - It is probably a good idea to copy (at least) some of your existing
+        watson frames into your new development folder, to test your changes:
+      
+            $ cp -R ~/.config/watson ./data/
+      
+        To make these used by watson, it is necessary to set the `WATSON_DIR`
+        variable. It can be set with:
+      
+            $ export WATSON_DIR=$PWD/data
+      
+        It may be a good idea to modify the `bin/activate` file from your virtualenv
+        (either `venv/bin/activate` or `$HOME/.virtualenvs/watson/bin/activate`),
+        and add the following lines (just before the `PATH` settings, for instance):
+        
+            WATSON_DIR=$PWD/data
+            export WATSON_DIR
+        
+        This will be set each time you start the virtualenv and you will not have to
+        re-set the variable each time.
 
 4.  Create a branch for local development:
 
         $ git checkout -b name-of-your-bugfix-or-feature
 
     Now you can make your changes locally.
-
+    
+    The files you need to edit to change watson's behavior are located in the
```

Comment:
Watson stands for the project, and `watson` for the command line tool:

```suggestion
    The files you need to edit to change Watson's behavior are located in the
```

Reply:
Sorry guys, I haven't been working on developing Watson for a while, due to other professional activities. I will resume and close this PR at the beginning of the next week.

Reply:
Don't worry, we are all volunteers here! :muscle:

Reply:
@jmaupetit would it be ok if I continue working on these in a different PR?

Reply:
I think you should ask @raphaelfournier first :wink:

Reply:
Of course! @raphaelfournier what do you think? can I help you close this PR somehow?

Reply:
No worries!